spec/requestReply_at_operation_with-filter-per-operation.yaml

```diff
@@ -0,0 +1,66 @@
+channels:
```

Comment:
I looked at `at_operation` files and decided to leave comments only here as it makes most sense for me and I like it the most (why not filtering per message? as we can always have multiple `null` channels with different messages, if it really happens that these reply headers might be different

Reply:
Agree. This is the solution that I like the most as well. Great stuff, @GreenRover. It really helped a lot seeing all these examples together 👏